src/cbor/tests/specific.rs

```diff
@@ -0,0 +1,1415 @@
+/*
+
+You can generate them from a set of JSON files with something like:
+
+```shell
+idx=1; for f in *.json ; do
+  symb=$(jq -r '.testCases[0].json.contents.contents.contents.error' "$f" | grep -Eo '([A-Z][a-z]*)+' | head -n 4 | tr '\n' '_' | sed -r 's/(^_+|_+$)//g')
+  cbor=$(jq -r '.testCases[0].cbor' "$f")
+  numb=$(printf '%04d' "$idx")
+  echo '#[tokio::test]'
+  echo '#[allow(non_snake_case)]'
+  echo 'async fn test_cbor_'"$numb"'_'"$symb"'() {'
+  echo '    verify_one("'"$cbor"'").await'
+  echo '}'
+  echo
+  idx=$((idx + 1))
+done
+```
+
+*/
+
+use super::verify_one;
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn test_cbor_223() {
+    verify_one("81820682820182008306001b000000024f522629820182008201d90102818258205176274bef11d575edd6aa72392aaf993a07f736e70239c1fb22d4b1426b22bc01").await
+}
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn test_cbor_222() {
```

Comment:
Some of these cases are humongous! 🐋 Maybe in the future it makes sense to move them to a separate `test_cbor_222.cbor` file, and only load the contents in this test function?

Reply:
@michalrus Yes, I agree.